flent/runners.py

```diff
@@ -1138,12 +1141,13 @@ def check(self):
 
         elif self.test == 'UDP_RR':
             self.units = 'ms'
+            args['host'] = ""
```

Comment:
This won't do the right thing if control_host is set...

Reply:
I don't see any tests that both use the UdpRttRunner (which sets the netperf test to UDP_RR) and also supports the --control-host option.

The netperf UDP_RR test only supports specifying the host via the global "-H" option.  netperf fails with "Invalid Argument" when the test-specific "-H" option is specified.

Can you give an example of a flent command that fails?

Reply:
The tests don't need to do anything to support it, the option is loaded directly from the settings by the runner:
```python
        args.setdefault('control_host', self.settings.CONTROL_HOST or self.host)
```

so you'll need to do a `args['control_host'] = args['host']` somewhere (and also deal with `control_local_bind` vs `local_bind`). We should probably also log a message warning if `control_host` is actually set while running a UDP_RR test, instead of just silently changing behaviour...